projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/.eslintrc.js

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright 2021 VMware, Inc.
```

Comment:
Isn't 2023 applicable here?

Reply:
Well noticed; did a quick check and my impression is, we need to update the NOTICE.txt year accordingly.

Reply:
It seems we need 2021-2023. What are your thoughts on this?

Reply:
Agree.